docs/apache-airflow/issue-triage-process.rst

```diff
@@ -0,0 +1,195 @@
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+ ..   http://www.apache.org/licenses/LICENSE-2.0
+
+ .. Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+
+
+
+Issue reporting and resolution process
+======================================
+
+This document explains the issue tracking and triage process within Apache
+Airflow including labels, milestones, and priorities as well as the process
+of resolving issues.
+
+Labels
+''''''
+
+Since Apache Airflow uses Github Issues as the issue tracking system, the
+use of labels is extensive. Though issue labels tend to change over time
+based on components within the project, the majority of the ones listed
+below should stand the test of time.
+
+The intention with the use of labels with the Apache Airflow project is
+that they should ideally be non-temporal in nature and primarily used
+to indicate the following elements:
+
+**Kind**
+
+The “kind” labels indicate “what kind of issue it is”. The most
+commonly used “kind” labels are: bug, feature, documentation, or task.
+
+Therefore, when reporting an issue, the label of ``kind:bug`` is to
+indicate a problem with the functionality, whereas the label of
+``kind:feature`` is a desire to extend the functionality.
+
+There has been discussion within the project about whether to separate
+the desire for “new features” from “enhancements to existing features”,
+but in practice most “feature requests” are actually enhancement requests,
+so we decided to combine them both into ``kind:feature``.
+
+We recently introduced ``kind:task`` to categorize issues which are
+identified elements of work to be done, primarily as part of a larger
+change to be done as part of an AIP or something which needs to be cleaned
+up in the project.
+
+Issues of ``kind:documentation`` are for changes which need to be
+made to the documentation within the project.
+
+
+**Area**
+
+The “area” set of labels should indicate the component of the code
+referenced by the issue. At a high level, the biggest areas of the project
+are: Airflow Core and Airflow Providers, which are referenced by ``area:core``
+and ``area:providers``. This is especially important since these are now
+being released and versioned independently.
+
+There are more detailed areas of the project such as Scheduler, Webserver,
+API, UI, Logging, and Kubernetes, which are all conceptually under the
+“Airflow Core” area of the project.
+
+Similarly, some providers are broken out such as Apache (``provider/Apache``),
```

Comment:
Is broken out the proper way to describe it? I'm not native English speaker but this sounds a bit odd to me. Maybe split / divide are more suitable?

Reply:
tried again, please let me know if this is better.